src/AdminForm.php

```diff
@@ -2223,11 +2223,12 @@ public static function insertComponent(&$field, &$enabled, $settings, $create_fi
     }
     // Retrieve option list
     if ($field['type'] === 'select') {
+      //To-do: Make this as a default type in src/Fields.php.
+      $field['type'] = 'civicrm_options';
       if ($options = $utils->wf_crm_field_options($field, 'component_insert', $settings['data'])) {
```

Comment:
@KarinG Before this change, civicrm_option is only set on an element, if there are valid options found in civicrm. For empty options, a normal select widget is created with no option values leading to an error.

I can't think of a field where we would need to add a "select" element instead of our "civicrm_options" field. The latter can be displayed in checkboxes, select, radio etc so I think we should default it in src/Fields.php. Do you agree?

Reply:
Yes agreed 👍